src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+input {
+  outline: none;
+}
```

Comment:
Ok, looks like reseting of default styles, but still, better to use classes for styles